modules/ca/src/tools/caget.c

```diff
@@ -388,7 +388,7 @@ int main (int argc, char *argv[])
     int digits = 0;             /* getopt() no. of float digits */
 
     int nPvs;                   /* Number of PVs */
-    pv* pvs;                    /* Array of PV structures */
+    pv* pvs = NULL;             /* Array of PV structures */
```

Comment:
pvs is used with `calloc()`, so this is not strictly necessary, but it's better coding style, agreed.